src/one_dragon_qt/view/installer_interface.py

```
@@ -1,919 +1,196 @@
+        self.gamepad_btn.clicked.connect(self.on_gamepad_clicked)
+        layout.addWidget(self.gamepad_btn, alignment=Qt.AlignmentFlag.AlignHCenter)
 
-        return content_widget
+        layout.addStretch(3)
+
+        return widget
+
+    # -------------------- 安装页逻辑 -------------------- #
+
+    def on_install_btn_clicked(self) -> None:
+        """一键安装：手动点击后按顺序安装所有卡片。"""
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)